### PR TITLE
chore: project addition by bindusreekalivarapu 

### PR DESCRIPTION
**Pull request by Kalivarapubindusree**
## Related Issue

#1666 

## Description

blood management system project is added

## Screenshots

Blood_bank

### CODE REVIEW

**Review by priyankarpal — CHANGES_REQUESTED** [review]
change the PR title to `chore: project addition by your-user-name`

File: DB/projects.json [hunk 1]
```diff
@@ -3967,4 +3967,17 @@
...
+  "Social_media": {
+    "gitHub": "https://github.com/Kalivarapubindusree",
+  },
+  "Projects": [
+    {
+      "link": "https://github.com/Kalivarapubindusree/blood-and-plasma-management-system",
+      "title": "Blood_And_Plasma_Management_System",
+      "description": "blood management system project is an like overall blood management system in our hands, Register,Logins, adding Events and showing Interest  to go and donate the blood seraching the donar near by the searched location showing the total donorslist and total bloodbanks and videos and contact us page and updating user profile within the working database",
+      "tech": ["html", "css","reactjs"]
+    },
```
Comment (priyankarpal) [inline DB/projects.json:3981]: remove last comma `,`
```suggestion
    }
```

File: DB/projects.json [hunk 2]
```diff
@@ -3967,4 +3967,17 @@
   ]
   }
 ]
```
Comment (priyankarpal) [inline DB/projects.json:3969]:
```suggestion
  },

```

File: DB/projects.json [hunk 3]
```diff
@@ -3967,4 +3967,17 @@
...
+  },
+  "Projects": [
+    {
+      "link": "https://github.com/Kalivarapubindusree/blood-and-plasma-management-system",
+      "title": "Blood_And_Plasma_Management_System",
+      "description": "blood management system project is an like overall blood management system in our hands, Register,Logins, adding Events and showing Interest  to go and donate the blood seraching the donar near by the searched location showing the total donorslist and total bloodbanks and videos and contact us page and updating user profile within the working database",
+      "tech": ["html", "css","reactjs"]
+    },
+  ]
+}
```
Comment (priyankarpal) [inline DB/projects.json:3983]: add `]` 
```suggestion
}
]
```

**Review by priyankarpal — CHANGES_REQUESTED** [review]

**Comment by Kalivarapubindusree** [comment]
Can you please check now @priyankarpal

**Review by priyankarpal — APPROVED** [review]

**Comment by Kalivarapubindusree** [comment]
Can you please add label for marks ?

**Comment by priyankarpal** [comment]
> Can you please add label for marks ? @priyankarpal This is issue resolved project can you please give marks @priyankarpal

We are not providing any labels for project additions as they are not associated with solving any issues.

**Comment by Kalivarapubindusree** [comment]
Ca you please tell me when you add labels for which type of contributions you add marks then??

**Comment by priyankarpal** [comment]
> Ca you please tell me when you add labels for which type of contributions you add marks then?? @priyankarpal

for only projects addition we're not providing any labels. But we will give labels to fixing any issue.

**Pull request merged**